src/components/table-hoc/TableRowNumberColumn.tsx

```diff
@@ -0,0 +1,11 @@
+import * as classNames from 'classnames';
+import * as React from 'react';
+import * as styles from './styles/TableRowNumber.scss';
+
+export const TableRowNumberColumn = (props: {number: string | number}) => (

```

Comment:
use ReactNode instead of a string or number

Reply:
Or simply use props.children?

Reply:
thanks, on it